docs/UsersGuide/source/Components.rst

```diff
@@ -0,0 +1,81 @@
+
+Output from UPP can be used with visualization, plotting, and verification packages or in
+further downstream post-processing (e.g., statistical post-processing techniques).
+
+Visualization Example
+=====================
+A Python script is provided to create basic visualization of the model output. The script
+is designed to output graphics in PNG format for 14 standard meteorological variables
+when using the pre-defined :term:`CONUS` domain. A difference plotting script is also included to visually compare two runs for the same domain and resolution. These scripts are provided only as an example for users familiar with Python. They may be used to perform a visual check to verify that the application is producing reasonable results. 
+
+After running ``manage_externals/checkout_externals``, the visualization scripts will be available in the ``ufs-srweather-app/regional_workflow/ush/Python`` directory. Usage information and instructions are described in :numref:`Chapter %s <Graphics>` and are also included at the top of the script. 
+
+Build System and Workflow
+=========================
+
+The SRW Application has a portable build system and a user-friendly, modular, and
+expandable workflow framework.
+
+An umbrella CMake-based build system is used for building the components necessary for running the end-to-end SRW Application, including the UFS Weather Model and the pre- and post-processing software. Additional libraries necessary for the application (e.g., :term:`NCEPLIBS-external` and :term:`NCEPLIBS`) are not included in the SRW Application build system but are available pre-built on pre-configured platforms. On other systems, they can be installed via the HPC-Stack (see :numref:`Chapter %s: Installing the HPC-Stack <InstallHPCStack>`). There is a small set of system libraries and utilities that are assumed to be present on the target computer: the CMake build software, a Fortran, C, and C++ compiler, and an :term:`MPI` library.
+
+Once built, the provided experiment generator script can be used to create a Rocoto-based
+workflow file that will run each task in the system in the proper sequence (see `Rocoto documentation
+<https://github.com/christopherwharrop/rocoto/wiki/Documentation>`_). If Rocoto and/or a batch system is not present on the available platform, the individual components can be run in a stand-alone, command line fashion with provided run scripts. The generated namelist for the atmospheric model can be modified in order to vary settings such as forecast starting and ending dates, forecast length hours, the :term:`CCPP` physics suite, integration time step, history file output frequency, and more. It also allows for configuration of other elements of the workflow; for example, whether to run some or all of the pre-processing, forecast model, and post-processing steps.
+
```

Comment:
Oh, this also just occurred to me - we will need to mention the ensemble capability in the UG as well. I think @JeffBeck-NOAA was going to work on that at some point! (Again, like the vx, that can be added later in a separate PR)

Reply:
@jwolff-ncar, yep, we'll need ensemble and stochastic physics included in the documentation.  I added some fairly detailed in-line comments within the original PRs, and those will be a good place to start.

Reply:
@JeffBeck-NOAA Can you link to the PR's and/or list the numbers I should look at?

Reply:
Yes!  [PR #237](https://github.com/ufs-community/regional_workflow/pull/237) and [PR #685](https://github.com/ufs-community/regional_workflow/pull/685) for the tendency-based and SPP stochastic physics, respectively.  There was also a bug fix for SPP with some documentation changes [here](https://github.com/ufs-community/regional_workflow/pull/704).